pierce-portfolio/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,40 +14,71 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
```

Comment:
nit: generally we don't add extra lines for import groupings within the same filepath